README.md

```diff
@@ -18,19 +18,20 @@ npm install after-all
 
 ```js
 var afterAll = require('after-all');
-var next = afterAll(done);
 
-setTimeout(next(function() {
-  console.log('Step two.');
-}), 500);
+var next = afterAll(function(err) {
+	if (err) throw err; // one of the asynchronous calls had an error
```

Comment:
Can you handle this error differently? I don't it's generally a good idea to do async throws, even if this is just an example. Other than this, looks nice. Thanks!